turbohare/hare.go

```diff
@@ -113,3 +114,8 @@ func (h *SuperHare) Start(ctx context.Context) error {
 func (h *SuperHare) Close() {
 	close(h.closeChannel)
 }
+
+// GetHareMsgHandler returns the gossip handler for hare protocol message.
+func (h *SuperHare) GetHareMsgHandler() pubsub.GossipHandler {
+	return nil
```

Comment:
i think it is better to return noop handler instead of nil

Reply:
done